UnityProject/Assets/Scripts/Core/Addressables/SingletonSOSounds.cs

```diff
@@ -23,6 +23,9 @@ public class SingletonSOSounds : SingletonScriptableObject<SingletonSOSounds>
    public AddressableAudioSource ExplosionCreak3 = null;
    public AddressableAudioSource Notice1 = null;
    public AddressableAudioSource Notice2 = null;
+   
+   public AddressableAudioSource GunEmptyAlarm = null;
+   public AddressableAudioSource KineticReload = null;
```

Comment:
Curious why `KineticReload` is in the singleton? Is it used more than just PKA?

Reply:
It's also used in the kinetic crusher, though admittedly that's only one other item.

Reply:
Alright.